tests/instruments/test_instrument.py

```diff
@@ -52,6 +159,28 @@ class Fake(Instrument):
     assert Fake.x.__doc__ == expected_doc
 
 
+def test_control_check_errors_get(generic):
+    generic.fake_ctrl_check_get_errors = True
+    generic.error = False
+
+    def checking():
+        generic.error = True
```

Comment:
shouldn't that be
```suggestion
    def checking(self):
        self.error = True
```
?

Reply:
That is, what I had tried first, but I got an error, that checking expected an argument, but got none.

Reply:
yes, because you're attaching it to an instance, not a class, so it does not get [bound automatically](https://stackoverflow.com/questions/972/adding-a-method-to-an-existing-object-instance).
Below invoke the "descriptor protocol" and do `generic.check_errors = checking.__get__(generic)`, should do the trick.